common/src/main/resources/assets/wynntils/lang/zh_tw.json

```diff
@@ -145,10 +145,10 @@
   "feature.wynntils.customNametagRenderer.name": "名稱標籤渲染",
   "feature.wynntils.customNametagRenderer.showGearOnHover.description": "是否應該在懸停時在名稱標籤上方顯示玩家裝備名稱?",
   "feature.wynntils.customNametagRenderer.showGearOnHover.name": "懸停時顯示玩家裝備",
-  "feature.wynntils.dataStorage.dryBoxes.description": "Dry Boxes for Function variable",
-  "feature.wynntils.dataStorage.dryBoxes.name": "Dry Boxes",
-  "feature.wynntils.dataStorage.dryCount.description": "Dry Count for Function variable",
-  "feature.wynntils.dataStorage.dryCount.name": "Dry Count",
+  "feature.wynntils.dataStorage.dryBoxes.description": "函數變量乾燥盒",
+  "feature.wynntils.dataStorage.dryBoxes.name": "無神話物品箱數",
+  "feature.wynntils.dataStorage.dryCount.description": "函數變量乾燥數",
+  "feature.wynntils.dataStorage.dryCount.name": "無神話物品盒數",
```

Comment:
You might have swapped the descriptions and names of these four lines

Reply:
Ahh, I just translate the name part. I'll change the description later.